src/bypasses/lnk.js

```diff
@@ -0,0 +1,19 @@
+import BypassDefinition from './BypassDefinition.js';
+
+export default class Lnk extends BypassDefinition {
+  constructor() {
+    super();
+    // custom bypass required bases can be set here
+  }
+
+  execute() {
+    if (!window.location.href.includes('/go/')) {
+      window.location.href = window.location.href.replace('lnk.parts', 'lnk.parts/go');
+    }
+    setTimeout(500);
```

Comment:
setTimeout is not synchronous (no idea if it is in this context)

Reply:
You're right, it does nothing anyways.